Gemfile

```diff
@@ -94,7 +94,7 @@ gem 'wicked_pdf'
 gem 'wkhtmltopdf-binary'
 
 gem 'immigrant'
-gem 'roo', '~> 2.8.3'
+gem 'roo', github: "roo-rb/roo" # master is currently needed for Ruby 3.x (awaiting new release)
```

Comment:
🤓

Reply:
Hm, makes me wonder if we are a bit early to the party. This is additional work. Hopefully it pays off.

Reply:
Ruby 3 has been around for over a year. The maintainers of the `roo` gem have just been a bit slow on updating/releasing.